preferences.json

```diff
@@ -7,5 +7,6 @@
       "y" : 71
     }
   },
-  "contactListFilePath" : "data/contactlist.json"
+  "contactListFilePath" : "data\\contactlist.json",
+  "taskListFilePath" : "data\\tasklist.json"
```

Comment:
This can leave the unix users baffled, maybe we can maintain versioning for this file offline, on the Telegram group, and alerts when something changes so that people can update locally.

Reply:
Yup, I just uploaded one sample tasklist for everyone to sync to when they pull upstream. gitIgnore still does ignore everything in the /data/ folder

Reply:
Will change!